src/main/java/com/fasterxml/jackson/dataformat/xml/deser/XmlTextPropertyNameHolder.java

```diff
@@ -0,0 +1,6 @@
+package com.fasterxml.jackson.dataformat.xml.deser;
+
+public class XmlTextPropertyNameHolder {
```

Comment:
I think I'd want different name for this, or (if there's natural place) just add as constant in an existing class/interface. But not sure exactly where.